### PR TITLE
First release of Kruskal-Higman

### PR DESCRIPTION
**Pull request by DmxLarchey**

### CODE REVIEW

**Comment by palmskog** [comment]
I see the following issue in CI:
```
+ /builds/coq/opam/opam-root-5.0.0-2.1.2-sandbox/opam-init/hooks/sandbox.sh "build" "make" "-j2" (CWD=/builds/coq/opam/opam-root-5.0.0-2.1.2-sandbox/5.0.0/.opam-switch/build/coq-kruskal-higman.1.0)
- export TIMED
- export TIMED
- cp -f theories/implem/prop.v theories/base/base_implem.v
- cp -f theories/implem/type.v theories/base/base_implem.v
- cp: cannot create regular file 'theories/base/base_implem.v': File exists
- rm -f theories/_CoqProject
- make: *** [Makefile:18: type] Error 1
- make: *** Waiting for unfinished jobs....
- echo '-R . KruskalHigmanProp' > theories/_CoqProject
- cat theories/CoqProject >> theories/_CoqProject
```
I don't really understand what's going on here, but it seems the error is real.

File: released/packages/coq-kruskal-higman/coq-kruskal-higman.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+name: "Coq-Kruskal-Higman"
```
Comment (palmskog) [inline released/packages/coq-kruskal-higman/coq-kruskal-higman.1.0/opam:2]:
```suggestion
```

**Comment by DmxLarchey** [comment]
@palmskog I think I found the issue. I forgot to restrict the build to the `type` target and `-j2` was transmitted to `make build`.

File: released/packages/coq-kruskal-higman/coq-kruskal-higman.1.0/opam (hunk 1 above)
Comment (DmxLarchey) [reply released/packages/coq-kruskal-higman/coq-kruskal-higman.1.0/opam]: I did it on my side

**Comment by palmskog** [comment]
@DmxLarchey unfortunately, there is now the following error:
```
[ERROR] The compilation of coq-kruskal-higman.1.0 failed at "make -j2 ".
#=== ERROR while compiling coq-kruskal-higman.1.0 =============================#
# context              2.1.2 | linux/x86_64 | ocaml-base-compiler.4.09.0 | file:///builds/coq/opam/released
# path                 ~/opam-root-4.09.0-2.1.2-sandbox/4.09.0/.opam-switch/build/coq-kruskal-higman.1.0
# command              ~/opam-root-4.09.0-2.1.2-sandbox/opam-init/hooks/sandbox.sh build make -j2 
# exit-code            2
# env-file             ~/opam-root-4.09.0-2.1.2-sandbox/log/coq-kruskal-higman-5339-abfbd7.env
# output-file          ~/opam-root-4.09.0-2.1.2-sandbox/log/coq-kruskal-higman-5339-abfbd7.out
### output ###
# make: *** empty string invalid as file name.  Stop.
```

**Comment by DmxLarchey** [comment]
I will look into it but not today.

**Comment by DmxLarchey** [comment]
@palmskog I spotted the problem:

```
build: [
  [make "-j%{jobs}%" "type"]
]
```

Quotes were missing arround the target `type`. I do not know why (w/o quotes) the target is interpreted as _empty_. Is `type` a keyword in this JSON syntax ?

Anyway it seems to work now ;-)

**Comment by palmskog** [comment]
@DmxLarchey for the record, I think anything without quotes inside `build` is interpreted as an opam variable/command, and if there is no such command, then it's the empty string.

**Pull request merged**